### PR TITLE
Az parse xd

### PR DESCRIPTION
**Pull request by andjelx**

### CODE REVIEW

File: scripts/90-split-archive.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import os
+import sys
+import re
+from xdfile.utils import progress, iso8601, get_args, args_parser, open_output, parse_pathname
+from xdfile.utils import filetime
+import xdfile.utils
+from xdfile.metadatabase import xd_sources_row, xd_sources_header
+
+
+p = args_parser('process huge puzzles archive into separate .zip and create sources.tsv')
```
Comment (saulpw) [inline scripts/90-split-archive.py]: Should be in main()

File: scripts/90-split-archive.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
```
Comment (saulpw) [inline scripts/90-split-archive.py]: Usage: comment

File: scripts/bwh-split.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+echo "rm -rf bwh_zips/ && mkdir bwh_zips"
```
Comment (saulpw) [inline scripts/bwh-split.sh]: Convert this script into documentation.

File: xdfile/puz2xd.py [hunk 4]
```diff
@@ -21,10 +21,18 @@ def reparse_date(s):
...
+    s = s.replace('\xc2\x92', "'")
+    s = s.replace('\xc3\x82',"")
+    s = s.replace('\xc3\xa8',"è") # +A5. Crème de la crème ~ ELITE
+    s = s.replace('\xc2', " ") # Change rest ot 0xC2 to 0x20
+    s = s.replace('\xa0'," ")
+    s = s.replace('\xe0'," ")
     s = s.replace('\x93', '"')
     s = s.replace('\x94', '"')
-    s = s.replace('\x85', '...')
+    s = s.replace('\x85', '…')
```
Comment (saulpw) [inline xdfile/puz2xd.py]: prefer ascii ... here

Comment (andjelx) [inline xdfile/puz2xd.py]: Fixed

File: scripts/bwh-split.sh (hunk 3 above)
Comment (andjelx) [inline scripts/bwh-split.sh]: Created PROCESSING.md in root

File: scripts/90-split-archive.py (hunk 1 above)
Comment (andjelx) [inline scripts/90-split-archive.py]: Done

File: scripts/90-split-archive.py (hunk 2 above)
Comment (andjelx) [inline scripts/90-split-archive.py]: Done

**Pull request merged**